packages/venia-concept/package.json

```diff
@@ -10,14 +10,14 @@
     "@magento/venia-drivers": "src/drivers"
   },
   "scripts": {
-    "build": "yarn run clean && node ../pwa-buildpack/bin/buildpack load-env --core-dev-mode . yarn run validate-queries && yarn run build:prod",
+    "build": "yarn run clean && node ../pwa-buildpack/bin/buildpack load-env --core-dev-mode . && yarn run validate-queries && yarn run build:prod",
     "build:analyze": "yarn run clean && mkdir dist && webpack -p --profile --no-progress --env.mode production --json > dist/build-stats.json && webpack-bundle-analyzer dist/build-stats.json",
     "build:dev": " ",
-    "build:prod": "webpack -p --color --no-progress --profile --env.mode production",
+    "build:prod": "webpack --no-progress --env.mode production",
```

Comment:
Why can't we have color for prod :(

Reply:
We can, we just don't force it if the terminal doesn't support it. We also aren't doing profile measurements just to throw it away.